_posts/2020-03-19-wirvsvirus.md

```diff
@@ -0,0 +1,17 @@
+---
+site: freiburg
+title: "WirVsVirus Hackathon"
+tags: [hackathon]
+---
+
+The Federal Government of Germany is organising the [#WirVsVirus Hackathon](https://wirvsvirushackathon.org/).
```

Comment:
I think this should be the second paragraph

"The Federal Government....., and the GalaxyProject is here to help."